ariadne/src/Ariadne/Wallet/Face.hs

```diff
@@ -36,6 +37,13 @@ data AccountReference
   | AccountRefByIndex !Word32 !WalletReference
   | AccountRefByName !Text !WalletReference
 
+-- | Reference to an account inside a wallet.
+data LocalAccountReference
```

Comment:
Why can't we reuse `AccountReference`?

Reply:
Because `AccountReference` contains `WalletReference` (`AccountRefByIndex` and `AccountRefByName` constructors). I want to ensure that all accounts are from the same wallet. If I had a list of `AccountReference`s, I wouldn't statically know that all of them have the same `WalletReference` (I could only write comments about it).